core/src/main/java/org/infinispan/reactive/publisher/impl/ClusterPublisherManagerImpl.java

```diff
@@ -206,6 +206,8 @@ public void start() {
       // This excludes the keys from the various address targets
       Map<Address, Set<K>> keyTargets = determineKeyTargets(topology, keysToInclude, localAddress, segments, ctx);
 
+      log.fatal("keyTargets is: " + keyTargets);
```

Comment:
Should these log.fatal statements still be here? :fearful:

Reply:
Whoops forgot to remove it :)